lib/fastlane/plugin/semantic_release/actions/analyze_commits.rb

```diff
@@ -315,6 +315,20 @@ def self.available_options
             description: "To parse version number from tag name",
             default_value: '\d+\.\d+\.\d+'
           ),
+          FastlaneCore::ConfigItem.new(
+            key: :prevent_tag_fallback,
```

Comment:
How is this related to the feature?

Reply:
I think the ability to opt out of falling back to `vX.X.X` as the tag is crucial for monorepos, especially if tags like v1.0.2 was created on the repository for a certain package that has nothing to do with the mobile-app.

I have a repository that I have inherited with tags that already exist like `v1.0.4` and now I am creating the mobile app package and do not want it to start from that tag that has nothing to do with mobile, therefore I would like to opt out.